content/sensu-go/6.7/plugins/assets.md

```diff
@@ -413,37 +413,72 @@ However, because the dynamic runtime asset directory path is exposed to asset co
+Each asset page in Bonsai lists the asset's namespace and name.
+This example shows where the namespace and name for the [http-checks][22] dynamic runtime asset are located in Bonsai:
 
-`%SENSU_WINDOWS_POWERSHELL_CHECKS_PATH%/include/config.yaml`
+{{< figure src="/images/go/assets_reference/name_namespace_location_bonsai_asset_paths.png" alt="Bonsai page for the Sensu http-checks dynamic runtime asset showing the location of the asset namespace and name" link="/images/go/assets_reference/name_namespace_location_bonsai_asset_paths.png" target="_blank" >}}
 
-The Windows console environment interprets the content between the [paired `%` characters][44] as an environment variable name and will substitute the value of that [environment variable][45].
+In this example, the Bonsai namespace is `sensu` and the asset name is `http-checks`.
+The environment variable for the [http-checks][4] asset path is:
 
-{{% notice note %}}
-**NOTE**: The Sensu Windows agent uses `cmd.exe` for the check execution environment.
-For all other operating systems, the Sensu agent uses the Bourne shell (sh) and `${VARIABLE_NAME}` [shell syntax](https://www.digitalocean.com/community/tutorials/how-to-read-and-set-environmental-and-shell-variables-on-linux).
-{{% /notice %}}
+{{< code shell >}}
+SENSU_HTTP_CHECKS_PATH
+{{< /code >}}
 
 ### Token substitution for dynamic runtime asset paths
 
 The `assetPath` token subsitution function allows you to substitute the dynamic runtime asset's local path on disk, so you will not need to manually update your check or hook commands every time the asset is updated.
 
 {{% notice note %}}
 **NOTE**: The `assetPath` function is only available where token substitution is available: the `command` attribute of a check or hook resource.
-If you want to access a dynamic runtime asset path in a handler or mutator command, you must use the [environment variable](#environment-variables-for-dynamic-runtime-asset-paths).
+To access a dynamic runtime asset path in a handler or mutator command, you must use the [environment variable](#environment-variables-for-dynamic-runtime-asset-paths).
 {{% /notice %}}
 
-For example, you can reference the dynamic runtime asset [`sensu-windows-powershell-checks`][4] from your check or hook resources using either the environment variable or the `assetPath` function:
+#### Linux environments
+
+The Linux environment interprets the content between the `${` and `}` characters as an environment variable name and will substitute the value of that environment variable.
+
+For example, you can use either the environment variable or the `assetPath` function to reference the dynamic runtime asset [`http-checks`][22] in your check or hook resources:
+
+{{< language-toggle >}}
+
+{{< code shell "Environment variable" >}}
+${SENSU_HTTP_CHECKS_PATH}/include/config.yaml
+{{< /code >}}
+
+{{< code shell "assetPath" >}}
+${{assetPath "sensu/http-checks"}}/include/config.yaml
```

Comment:
I still need to confirm whether the assetPath should include the Bonsai namespace